openmdao/utils/tests/test_notebook_utils.py

```diff
@@ -0,0 +1,41 @@
+""" Unit tests for the notebook_utils."""
+
+import unittest
+try:
+    from tabulate import tabulate
+except ImportError:
+    tabulate = None
+
+try:
+    import IPython
+except ImportError:
+    IPython = False
+
+import openmdao.api as om
+from openmdao.utils.assert_utils import assert_warning
+
+@unittest.skipUnless(tabulate and IPython, "Tabulate and IPython are required")
+class TestNotebookUtils(unittest.TestCase):
+
+    def test_show_options_wo_attr(self):
+        from openmdao.utils import notebook_utils
+        notebook_utils.ipy = True
+
+        options = om.show_options_table("openmdao.components.cross_product_comp.CrossProductComp")
+
+        self.assertEqual(options, None)
+
+    def test_show_options_w_attr(self):
+        from openmdao.utils import notebook_utils
```

Comment:
It's not clear to me what the difference is between this test and the previous one.  Maybe put some comments in here to explain what these tests are doing.

Reply:
I missed the declared options in the base component so there is no difference. Deleted

Reply:
It doesn't look like you are actually testing the new functionality? `BalanceComp` has an `options` dict, right?  
Don't you need an object ```where the options are held in the __init__ class instead of an options dict.```?

Reply:
I added a better test for a case where the options dictionary is being passed directly